app/views/reports/regions/_index_tree.html.erb

```diff
@@ -0,0 +1,75 @@
+<div id="facility-access" data-page="show">
+
+  <% regions.each do |org, regions_by_district| %>
+
+  <div class="card access-tree show">
+    <div class="access-tree__section-header">
+      <%= org.name %>
+    </div>
+
+    <ul>
+      <% regions_by_district.each do |district, regions_by_block| %>
```

Comment:
Descend through the tree, and show the accessible _or_ just the parent regions of things that the user may have access to.